api/src/test/resources/org/openmrs/module/xstream/include/PersonAttributeTypeShortSerializationTest.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <dataset>
-    <person_attribute person_attribute_id="1" person_id="501" value="" person_attribute_type_id="1" creator="1" date_created="2008-08-15 15:46:47.0" voided="false" uuid="0768f3da-b692-44b7-a33f-abf2c450474e"/>
-    <person_attribute_type person_attribute_type_id="1" name="Race" description="Group of persons related by common descent or heredity" format="java.lang.String" searchable="false" creator="1" date_created="2007-05-04 09:59:23.0" retired="true" retired_by="1" date_retired="2008-08-15 00:00:00.0" retire_reason="test" uuid="b3b6d540-a32e-44c7-91b3-292d97667518" sort_weight="1"/>
```

Comment:
Was this simply formatting? It is hard to differentiate between the actual changes and formatting.